OpenRA.Mods.Common/Traits/AffectsShroud.cs

```diff
@@ -153,23 +174,33 @@ void INotifyRemovedFromWorld.RemovedFromWorld(Actor self)
 				RemoveCellsFromPlayerShroud(self, p);
 		}
 
-		public virtual WDist Range => CachedTraitDisabled ? WDist.Zero : Info.Range;
+		public virtual WDist Range => IsTraitDisabled ? WDist.Zero : Info.Range;
 
-		void INotifyMoving.MovementTypeChanged(Actor self, MovementType type)
+		readonly struct Input : IEquatable<Input>
```

Comment:
maybe a more descriptive name? Or a summary?

Reply:
Renamed to `InputPositions`.